aii-ks/src/main/perl/ks.pm

```diff
@@ -1614,7 +1614,7 @@ function join { local IFS=","; echo "\$*"; }
 if [ -x \$efibootmgr ]; then
     ordertxt=\$(\$efibootmgr |grep BootOrder | sed "s/.*: \\?//")
     order=(\$(echo \$ordertxt | tr ',' ' '))
-    pxe=(\$(\$efibootmgr -v | grep -ie ' \\(NIC\\|PXE\\|Network\\) ' | sed "s/*\\? .*//;s/^Boot//" | sort ))
+    pxe=(\$(\$efibootmgr -v | grep -Pi '\s+(NIC|PXE|Network)\s+' | sed "s/*\\? .*//;s/^Boot//" | sort ))
```

Comment:
no need for the `+`s

and is this tested? this is part of a heredoc, you probably need to double backslashes, so `\\s` instead of `\s`